pages/identities.tsx

```diff
@@ -93,6 +68,11 @@ const Identities: NextPage = () => {
     //   });
   }, [account, router.asPath]);
 
+  useEffect(() => {
+    if (!mintData?.transaction_hash) return;
+    addTransaction({ hash: mintData?.transaction_hash ?? "" });
```

Comment:
I am not sure the ?? is required since you check if the txhash has a value before